setup.py

```diff
@@ -21,13 +21,10 @@ def readfile(filename):
     'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     'Topic :: Software Development :: Libraries :: Application Frameworks',
     'Development Status :: 5 - Production/Stable',
-    'Programming Language :: Python :: 3.7',
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
     'Framework :: Django',
-    'Framework :: Django :: 2.2',
-    'Framework :: Django :: 3.2',
```

Comment:
Well, one feature doesn't work in Django-3.2 and lower. This is only available with the new API introduced in Django-4, otherwise I'd have to extend both, the Stacked- and the Tabular-Inline templates.

One of the benefits of upcoming version 2 is that we don't have to extend any templates anymore.

Therefore, those who want to stick with Django-3.2 shall use version 1.0.4.

Reply:
Ah now that makes sense 👍